osversion_android.go

```diff
@@ -1,87 +1,11 @@
 package osversion
 
-/*
```

Comment:
We're using this code in production right @oxtoacart? @atavism do we need the osversion on Android? Is that used?